GNUmakefile

```diff
@@ -17,18 +17,12 @@ PROTOC_ZIP := protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-x86_64.zip
 PROTOC_URL := https://github.com/protocolbuffers/protobuf/releases/download/v$(PROTOC_VERSION)/$(PROTOC_ZIP)
 PROTOC_ROOT := .protobuf/protoc-$(PROTOC_OS)-$(PROTOC_VERSION)
 PROTOC_BIN := $(PROTOC_ROOT)/bin/protoc
-GOPROTOVERSION?=$(shell grep github.com/golang/protobuf go.mod | awk '{print $$2}')
-GOPROTOTOOLS = \
-	github.com/golang/protobuf/protoc-gen-go@$(GOPROTOVERSION) \
-	github.com/hashicorp/protoc-gen-go-binary@master \
-	github.com/favadi/protoc-go-inject-tag@v1.3.0 \
-	github.com/hashicorp/mog@v0.2.0
 
 GOTAGS ?=
 GOPATH=$(shell go env GOPATH)
 MAIN_GOPATH=$(shell go env GOPATH | cut -d: -f1)
 
-export PATH := $(PWD)/bin:$(PATH)
+export PATH := $(PWD)/bin:$(GOPATH)/bin:$(PATH)
```

Comment:
Having only ever used Go modules, my `GOPATH` env var is empty, but `go install` puts things into `~/go/bin`, not sure if there's a better way to grab that default.

Reply:
It'll work just fine for you:
```
$ GOPATH= go env GOPATH     # unset whatever i have configured (this is what the makefile does on line 28 above)
/home/rboyer/go             # same as you
```